MagickCore/nt-base.h

```diff
@@ -99,7 +103,9 @@ extern "C" {
 #if !defined(closedir)
 #  define closedir(directory)  NTCloseDirectory(directory)
 #endif
-#define MAGICKCORE_HAVE_ERF
+#if !defined(MAGICKCORE_HAVE_ERF)
```

Comment:
Why do we need this?

Reply:
Same as above, it is defined by default in mingw-w64 (perhaps via configure)?

Reply:
I have applied these changes with a separate commit.